src/teuthology_api/routes/kill.py

```diff
@@ -28,5 +29,16 @@ def create_run(
     or else it will SyntaxError: non-dafault
     argument follows default argument error.
     """
-    args = args.model_dump(by_alias=True, exclude_unset=True)
-    return run(args, logs, access_token, request)
+    try:
+        args = args.model_dump(by_alias=True, exclude_unset=True)
+        return await run(args, logs, token, request)
+    except HTTPException:
```

Comment:
should we log the exception here as well? just to keep it consistence with lines below